test/e2e/e2e_flags.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"flag"
+)
+
+// ClientFlags saves the environment flags for client.
+var ClientFlags = initializeFlags()
```

Comment:
How does doing this fix the CI flow errors? Is there something missing in this PR? Or perhaps elaborate the problem and fix. Thx

Reply:
`emitmetrics` is a flag used by our testing framework, which is also *required* for the running of periodic jobs. But since `Knative client` is not using the framework, it does not have the flag, so adding this `flag` as a no-op can solve the error.
I have made it simpler and added more meaningful comments.

Reply:
Not obvious but thanks for the explanation. Hopefully fixes things. Thanks.